ios/.xcode.env

```diff
@@ -9,4 +9,3 @@
 # For example, to use nvm with brew, add the following line
 # . "$(brew --prefix nvm)/nvm.sh" --no-use
 export NODE_BINARY=$(command -v node)
-export RCT_NEW_ARCH_ENABLED=false
```

Comment:
Revert me, see l.7 in `Podfile`, we want this to be explictly false to keep things declarative and avoid magic

Reply:
So I think we need to add it to `Podfile.properties.json` instead of there as it should be `'1'` and not `'true'`